README.md

```diff
@@ -81,6 +81,13 @@ The **Add Bookmark** form allows you to add new bookmark with name, link and com
 
 When moving your mouse over an existing bookmark, you can either click *recycle icon* to delete it or *pencil icon* to edit it.  The editing action works just in the same way as adding a new bookmark.
 
+### Browser Integration
+
+For the most of popular web browsers you can install a [userscript](https://en.wikipedia.org/wiki/Greasemonkey) from [here](add-bookmark.user.js).
+
+It will add userscript menu commands to setup URL of your `index.html` bookmark manager file and to bookmark current page.
+Last one will open bookmark manager with new bookmark title and URL filled in � very helpful.
+
```

Comment:
i.e.  This just needs a little extra detail.  Something like:

    The intention is that when you click on your bookmark the `add bookmark` area will be populated with the appropriate link and title, such that it can be added quickly and easily.
    This will work if your bookmarks are hosted remotely, and accessed via a `http` or `https` URL, but unfortunately
     if you're accessing locally, via a `file://`-based prefix then it will not work.

I don't know, it'll need a bit of explanation anyway.